addons/l10n_in/views/report_invoice.xml

```diff
@@ -58,6 +58,59 @@
             </h2>
         </xpath>
 
+        <xpath expr="//div[@id='payment_term']" position="before">
```

Comment:
This seems to be adding a completely new feature?
More details in commit message ~~plz~~ please

Reply:
@william-andre Yes. I'm awaiting the fw-port of this feature from
https://github.com/odoo/odoo/pull/155662

... you are reviewing too quickly :man_facepalming: